pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -107,7 +105,20 @@ func (f *Fit) Filter(ctx context.Context, cycleState *framework.CycleState, pod
 		return framework.NewStatus(framework.Error, err.Error())
 	}
 	_, reasons, err := predicates.PodFitsResourcesPredicate(pod, r, f.ignoredResources, nodeInfo)
```

Comment:
It seems except for "legacy_registry.go", including this file, there are a few occurances depending on predicates package. We can cleanup them in this PR.

```
⇒  ag "\"k8s.io/kubernetes/pkg/scheduler/algorithm/predicates\"" pkg/scheduler/framework
pkg/scheduler/framework/plugins/legacy_registry.go
28:	"k8s.io/kubernetes/pkg/scheduler/algorithm/predicates"

pkg/scheduler/framework/plugins/nodevolumelimits/csi_test.go
37:	"k8s.io/kubernetes/pkg/scheduler/algorithm/predicates"

pkg/scheduler/framework/plugins/noderesources/fit.go
27:	"k8s.io/kubernetes/pkg/scheduler/algorithm/predicates"

pkg/scheduler/framework/plugins/noderesources/fit_test.go
31:	"k8s.io/kubernetes/pkg/scheduler/algorithm/predicates"
```

Reply:
What I want to do in a followup PR is move PodFitsResourcesPredicate to noderesources/fit.go, and modify its signature to return a list resources that caused the failure, Kubelet depends on that.

Reply:
cleaned it up in this PR.